src/test/java/racingcar/game/RacingGameTest.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.game;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.domain.CarDTO;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingGameTest {
+
+    static RacingGame racingGame = new RacingGame();
+    static List<List<CarDTO>> totalExecutionResult;
+
+    @BeforeAll
+    public static void setRacingGame() {
+        racingGame.initCarNames("woo,te,co");
+        racingGame.initTotalAttempt("5");
+
+        totalExecutionResult = racingGame.play();
+    }
```

Comment:
BeforeEach를 하는 게 낫지 않을까요?

```suggestion
    private RacingGame racingGame;
    private List<List<CarDTO>> totalExecutionResult;

    @BeforeEach
    public void setRacingGame() {
        racingGame = new RacingGame();
        racingGame.initCarNames("woo,te,co");
        racingGame.initTotalAttempt("5");
        
        totalExecutionResult = racingGame.play();
    }
```

Reply:
해당 메서드는 전체 테스트 전 단 한번만 실행되어도 충분할 것 같아 BeforeAll을 사용했습니다!